SU-SWS/stanford_profile_helper
# D8CORE-3686: Intranet styles.

# READY FOR REVIEW
Resolved questions
- Favicon - added in the Theme settings as part of the site set up.
- Admin site - styles excluded.

# Summary
- Intranet styles
- Styles should not be included on the /admin/ pages.

# Needed By (Date)
- 4/7

# Urgency
- High

# Steps to Test

1. Pull in this change.
2. Clear caches.
3. Verify the following:
   -  Brand bar is black
   - Body color is #f4f4f4
   - local footer is #eaeaea
   - super footer is #ffffff
   - Add content to the 3rd column in the Local footer. First `<p>` should say "Intranet" which will give you the bold and the icon. The second `<p>` would be the disclaimer.

# Affected Projects or Products
- Intranet project

# Associated Issues and/or People
- D8CORE-3686

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
